ivre/db/mongo.py

```diff
@@ -182,6 +182,12 @@ def migrate_schema(self, colname, version):
             updated = False
             new_version, migration_function = self.schema_migrations[
                 colname][version]
+            if config.DEBUG:
```

Comment:
Agreed, `logging` should be used everywhere, but that's another job! Maybe you could open an issue to track this?